.ci.yaml

```diff
@@ -3244,7 +3244,7 @@ targets:
     timeout: 60
     properties:
       tags: >
-        ["devicelab","ios","mac"]
+        ["devicelab","hostonly","ios","mac"]
```

Comment:
Since this run on a `Mac_ios` target, it'll still run in the devicelab. We'd need to add a separate target, like `Mac ios_app_with_extensions_test`.

Reply:
Thanks @CaseyHillers.  I'll just revert this change for now since I don't want to try to figure out how to do that on the .ci.yaml roller failures.